src/0xsquid/v1/route.ts

```diff
@@ -248,18 +249,29 @@ export const parseParams = (data: any): RouteParams => {
   });
 };
 
-export const parseRouteResponse = (response: any): RouteResponse => {
+export const parseRouteResponse = (
+  response: any,
+  headers: AxiosResponseHeaders
+): RouteResponse => {
   const {
     route: { estimate, transactionRequest, params }
   } = response;
-  const routeResponse = {
+  const routeResponse = removeEmpty({
+    requestId:
```

Comment:
requestId and integratorId is implemented twice in parseRouteResponse and parseStatusResponse
maybe refactor.

Reply:
I've extracted the logic in a separate method